cypress-tests/cypress/e2e/v2/verifyInfrastructure.cy.ts

```diff
@@ -0,0 +1,81 @@
+/**
+ * Retry invoking a Cypress request until it succeeds or fail a few times in a row.
+ * @param url - target URL
+ * @param service - name of the tested service
+ * @param limit - maximum attempts before failing
+ * @param delaySeconds - delay between attempts
+ * @returns response body from the request or `null`
+ */
+function retryRequest(
+  url: string,
+  service: string,
+  limit = 6,
+  delaySeconds = 21,
```

Comment:
why 21?

Reply:
What's wrong with 21? I changed it to 20 so it's a round number, if that's what you meant. Or did you mean it should be much bigger/smaller?